### ISSUE TITLE
image naming and `devspace dev`

### ISSUE THREAD

**Issue by tomasc** [devspace-cloud/devspace#998]
**What happened?**  
When I change the image name in the `devspace.yml` from `default` to anything else the `devspace dev` command fails with `Unable to start portforwarding …`.

**What did you expect to happen instead?**  
Start in the dev mode.

**How can we reproduce the bug?** (as minimally and precisely as possible)  
Change the imageName `default` to anything else in all 3 instances:

```yaml
version: v1beta6

images:
  default:
    image: […]

deployments:
[…]

dev:
  ports:
  - imageName: default
  […]
  sync:
  - imageName: default
  […]
```

**Local Environment:**  
- DevSpace Version: 4.6.0
- Operating System: mac
- Deployment method: helm

**Kubernetes Cluster:**  
- Cloud Provider: google | aws | azure | DevSpace Cloud | DevSpace Cloud Enterprise | other
- Kubernetes Version: [use `kubectl version`]

**Anything else we need to know?**  
Output of `devspace dev -i`:

```
Successfully built bfa9d4b229a4
Successfully tagged ???/???:USUQjGI
[info]   Skip image push for ???/???
[done] √ Done processing image '???/???'
[info]   Skipping deployment database
[done] √ Deployed helm chart (Release revision: 0)
[done] √ Successfully deployed ??? with helm
[fatal]  Unable to start portforwarding: Error selecting pod: Couldn't find a running pod with image selector '???/???:USUQjGI'

kubectl get pods
NAME                   READY   STATUS             RESTARTS   AGE
???-6988d9b5dd-x7hhk   0/1     ImagePullBackOff   0          5m11s
```

(PS replaced my Docker image name with `???`)

/kind bug

**Comment by FabianKramm** [comment]
@tomasc thanks for reporting this issue!

**Comment by FabianKramm** [comment]
Fixed by #999

**Comment by tomasc** [comment]
@FabianKramm superfast, thanks!
